bluebottle/wallposts/models.py

```diff
@@ -209,6 +209,22 @@ class Reaction(models.Model):
     objects = ReactionManager()
     objects_with_deleted = models.Manager()
 
+    class Analytics:
+        type = 'wallpost'
+        tags = {}
+        fields = {
+            'id': 'id',
+            'user_id': 'author.id'
+        }
+
+        def extra_tags(self, instance, created):
+            return {'sub_type': 'reaction'}
+
+        def skip(self, obj, created):
```

Comment:
Probably don't need this either. The reactions are always from users so even if the reaction is to  system wallpost we should record the event.